db/seeds.rb

```diff
@@ -18,18 +27,12 @@
     user_id: 1,
     commute_id: rand(1..Commute.all.length),
     comment: Faker::Hobbit.quote,
-    tally_count: 2
+    review_stars: 2
   )
 end
 
 
-# 8.times do
-#   User.create(
-#     username: Faker::Ancient.god,
-#     first_name: Faker::Name.first_name,
-#     last_name: Faker::Name.last_name
-#   )
-# end
+
 
 # 12.times do
```

Comment:
Remove any commented out code.

Additionally, while not required, I would highly recommend avoiding Faker for seeds and actually taking the time to put in good examples, but it is up to you!